config.json.template

```diff
@@ -1,4 +1,5 @@
 {
   "api_key" : "",
-  "bot_name" : ""
+  "bot_name" : "",
+  "send_help_as_dm" : "1"
```

Comment:
Since `send_help_as_dm` is a boolean, shouldn't it be `"send_help_as_dm" : true` ?

Reply:
Had it boolean first, but parsing in console seems easier when only using strings.